frontend/components/TodoSearch.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, IconButton, TextField } from "@mui/material";
+import { ChangeEvent, useEffect, useState } from "react";
+import SearchIcon from "@mui/icons-material/Search";
+import { TodoSearchProps } from "@/types";
+
+const TodoSearch: React.FC<TodoSearchProps> = ({ notes, setFilteredNotes }) => {
+  const [search, setSearch] = useState<string>("");
```

Comment:
```suggestion
  const [search, setSearch] = useState("");
```
тип берётся из значений, нет смысл писать явно тип, если он будет такой же как и поставит сам useState

Reply:
поняла, поправила